interpreter/src/main/scala/pl/touk/nussknacker/engine/compile/nodecompilation/ValueEditorValidator.scala

```diff
@@ -0,0 +1,84 @@
+package pl.touk.nussknacker.engine.compile.nodecompilation
+
+import cats.data.Validated.{Valid, invalidNel}
+import cats.data.ValidatedNel
+import cats.implicits.toTraverseOps
+import pl.touk.nussknacker.engine.api.context.PartSubGraphCompilationError
+import pl.touk.nussknacker.engine.api.context.ProcessCompilationError._
+import pl.touk.nussknacker.engine.api.definition.{
+  DualParameterEditor,
+  FixedExpressionValue,
+  FixedValuesParameterEditor,
+  ParameterEditor
+}
+import pl.touk.nussknacker.engine.api.editor.DualEditorMode
+import pl.touk.nussknacker.engine.graph.node.FragmentInputDefinition.{FragmentClazzRef, ValueInputWithFixedValues}
+
+object ValueEditorValidator {
+
+  def validateAndGetEditor( // this method doesn't validate the compilation validity of FixedExpressionValues (it requires validationContext and expressionCompiler, see FragmentParameterValidator.validateFixedExpressionValues)
+      valueEditor: ValueInputWithFixedValues,
+      initialValue: Option[FixedExpressionValue],
+      refClazzName: Option[String],
```

Comment:
1. Let's use value class (`FragmentClazzRef`) instead of `String`
2. Let's extract two methods - one which validates against `clazzRef` and another which validates rest. Thanks to that we avoid passing None for components. Also, I think that the first method (`validateAgainstClazzRef`) should be in `FragmentParameterValidation` because for components it is impossible to use it yet.